nomos-services/storage/src/api/da/mod.rs

```diff
@@ -1,9 +1,43 @@
 use std::{collections::HashSet, error::Error};
 
 use async_trait::async_trait;
+use nomos_core::da::blob::Share;
 
 pub mod requests;
 
+pub trait DaConverter<B: StorageDaApi> {
```

Comment:
Wasn't the new convention to avoid single letter generics? Cc @danielSanchezQ

Reply:
Not sure about the new conversion. But we should avoid (in fact is forbidden) to use single letter generics anymore. And if working on something there are legacy single letter ones, lets please just refactor them in place. Is a common effort.